### DISCUSSION TITLE
Sumatra 3.4.6 64 bit -  Cannot OPEN Attached/embebed PDF files

### DISCUSSION THREAD

**Discussion by sabimh** [sumatrapdfreader/sumatrapdf#3040] [Documentation]
Hi. I am using Sumatra 3.4.6 64 bit, and I can see BUT not open Attached/embebed PDF files in one PDF.
Check example image
Can you assist me?
![2022-10-21 11_23_46-Configuración](https://user-images.githubusercontent.com/91084069/197219054-91c2397c-870f-41b7-b032-af2951be6662.jpg)

**Comment by GitHubRulesOK** [comment]
It depends how it was attached, version 3.1.2 could extract and save inline attachments (3.4# /3.5# cannot due to MuPdf engine changes)
![image](https://user-images.githubusercontent.com/12754913/197231814-bfcc33b3-8436-4d5f-b2f2-7f358e4c1fd4.png)
it could also view annotation embedment's, here click on paperclip will run as that second tab (3.4x 3.5# cannot)
![image](https://user-images.githubusercontent.com/12754913/197236920-af896f07-7aad-47d7-9760-43f2181f0f38.png)
